nibabel/nicom/ascconv.py

```diff
@@ -0,0 +1,60 @@
+# emacs: -*- mode: python-mode; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Parse the "ASCCONV" meta data format found in a variety of Siemens MR files.
+"""
+import ast, re
+from ..externals import OrderedDict
+
+
+ASCCONV_RE = re.compile(
+    r'### ASCCONV BEGIN((?:\s*[^=\s]+=[^=\s]+)*) ###\n(.*?)\n### ASCCONV END ###',
+    flags=re.M | re.S)
+
+
+def parse_ascconv(csa_key, ascconv_str):
```

Comment:
Maybe it would be more obvious what is going on if we had

```
parse_ascconv(ascconv_str, str_delim):
```

Then we put the logic for which string delimiter per protocol somewhere else?   The `csa_key` is just signaling the string delimiter at the moment.